ci/docker_prune.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -i
 
-echo "Removing containers older then 8 hours"
-docker container prune -f --filter "until=8h"
+echo "Removing containers older then 1 hours"
+docker container prune -f --filter "until=1h"
```

Comment:
I am not against this change, but i think it is un-related from the issue we have. This only cleans up containers that have been stopped and are at least X hours old. 

I would suggest to leave this be to keep the changes as minimal, and we haven't seen any issue with the 8hr strategy.

Reply:
Actually yea, this is a good point. This may also mask if the trap doesn't work ... :)